tests/contrib/aredis/test_aredis.py

```diff
@@ -201,11 +201,12 @@ async def test_opentracing(tracer, snapshot_context):
 @pytest.mark.subprocess(env=dict(DD_REDIS_RESOURCE_ONLY_COMMAND="false"))
 @pytest.mark.snapshot
 def test_full_command_in_resource_env():
+    import ddtrace.auto  # noqa
+
     import asyncio
 
     import aredis
 
-    import ddtrace
```

Comment:
Why doesn't this break uses of `ddtrace` below? I'm definitely just missing something but I want to understand.

Reply:
Ah well spotted. I think this was definitely broken.